client/src/Edit/elements/ReferenceInput.tsx

```diff
@@ -130,24 +130,16 @@ export default class ReferenceInput extends Component<Props, State> {
 
   fetchItems = opts => {
     const { type, models } = this.props;
-    const [firstModel, ...otherModels] = models;
 
-    const queryString = query({
+    const queryString = stringify({
       ...opts,
       linkable: models.length === 0,
-      models: otherModels && otherModels.length ? models : undefined
+      models: models.length ? models : undefined
     });
 
-    // if reference has only one options, use list instead of search
-    return api
-      .get(
-        `/${type}${
-          models.length === 0 || otherModels.length > 0 ? "" : `/${firstModel}/`
-        }${queryString}`
-      )
-      .then(res => {
-        if (res) this.setState({ items: res.items });
-      });
+    return api.get(`/${type}?${queryString}`).then(res => {
```

Comment:
Can queryString be empty? I would like to not have dangling `?` in these cases.

Reply:
No, it can't be empty!